src/window.rs

```diff
@@ -1115,6 +1115,31 @@ impl Window {
     self.window.set_progress_bar(_progress)
   }
 
+  /// Sets the badge count on the taskbar
+  ///
+  /// ## Implementation Details
+  /// - _desktop_filename is for linux ONLY!
+  /// 
+  /// ## Platform-specific
+  ///
+  /// - **Windows**: Windows allows to set an arbitrary .ico file. See set_overlay_icon 
+  /// - **Linux / macOS**: Badge count is app-wide and not specific to this window. Only supported desktop environments with `libunity` (e.g. GNOME).
+  /// - **iOS / Android:** Unsupported.
+  #[inline]
+  pub fn set_badge_count(&self, _count: Option<i64>, _desktop_filename: Option<String>) {
```

Comment:
this is no longer needed since we have platform-specific methods on WindowExtMacos, WindowExtWindows and WindowExtUnix

Reply:
Yeah, 😅 I forgot to remove that